src/common/base-store.ts

```diff
@@ -111,10 +111,11 @@ export class BaseStore<T = any> extends Singleton {
   }
 
   protected onSync(model: T) {
-    // todo: use "resourceVersion" if merge required (to avoid equality checks => better performance)
-    if (!isEqual(this.toJSON(), model)) {
-      this.fromStore(model);
-    }
+    this.applyWithoutSync(() => {
```

Comment:
Prevent syncing model changes back to caller

Reply:
is the todo no longer applicable?

Reply:
I think that is still valid. Will bring it back